app/views/manageBusinesses/ManageYourBusinesses.scala.html

```diff
@@ -80,9 +80,9 @@
 
 @addBusinessNameURL = @{
```

Comment:
does this work?
`
@addBusinessNameURL = @{
        controllers.manageBusinesses.add.routes.AddBusinessNameController.show(isAgent = isAgent, isChange = false).url
`